CHANGELOG.md

```diff
@@ -4,6 +4,154 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.45.0] - 2024-04-22
+
+## 🚀 Features
+
+### Query validation process with Rust ([PR #4551](https://github.com/apollographql/router/pull/4551))
+
+The router has been updated with a new Rust-based query validation process using `apollo-compiler` from the `apollo-rs` project. It replaces the Javascript implementation in the query planner. It improves query planner performance by moving the validation out of the query planner and into the router service, which frees up space in the query planner cache. 
+
+Because validation now happens earlier in the router service and not in the query planner, error paths in the query planner are no longer encountered. The new error messages should be clearer.
```

Comment:
I'm not sure how I'd write this, but I would like it to be explicit that users may see some *changed* error messages (which is currently implied by saying "new error messages", but still)

Maybe even
```suggestion
Because validation now happens earlier in the router service and not in the query planner, error paths in the query planner are no longer encountered. Some messages in error responses to invalid queries are now clearer.
```
to keep it positive but to emphasise that there is a change.

not a dealbreaker i think anyway.

Reply:
I applied a small variation of your suggestion.  Thank you!